README.md

```diff
@@ -25,9 +25,26 @@ MQ:
   port: <MQ Port>
   server: <MQ Hostname or IP>
   users:
-      <LLM MQ service_name>:
+    <LLM MQ service_name>:
       user: <MQ user>
       password: <MQ user's password>
   LLM_<LLM NAME uppercase>:
-      num_parallel_processes: <integer > 0>
+    num_parallel_processes: <integer > 0>
 ```
+
+## Enabling Chatbot personas
+An LLM may be configured to connect to a `/chatbots` vhost and participate in
+discussions as described in the [chatbots project](https://github.com/NeonGeckoCom/chatbot-core).
+One LLM may define multiple personas to participate as:
+```yaml
+llm_bots:
+  <LLM Name>:
+    - name: Assistant
+      description: You are a personal assistant who responds in 40 words or less
```

Comment:
From `persona-private`:
```yaml
  short_response: True  # If true, limit to 60 words
  base_prompt: You are a smart city manager...  # Base personality prompt to which response length can be added
  imagination: False  # If true, allow for non-factual information in responses
```

Reply:
Leave the internals to `name` and `description`; UI can accept these other params but will generate a `description` to be stored